gef.py

```diff
@@ -1636,6 +1640,7 @@ def print_call_args(self):
 
         return
 
+
```

Comment:
extra line (We only put two lines between top level functions and classes, not methods)